sdk/spring/ci.yml

```diff
@@ -82,3 +85,7 @@ extends:
         groupId: com.azure
       - name: azure-spring-boot-sample-storage-blob
         groupId: com.azure
+      - name: azure-spring-data-sample-gremlin
```

Comment:
This artifact name looks odd to me. Is azure-spring-boot different from azure-spring-data? I'm asking because above you have azure-spring-boot-sample-data-gremlin and here we have azure-spring-data-sample-gremlin, shouldn't this be ending with sample-data-gremlin?

Reply:
Yes, azure-spring-boot is different from azure-spring-data.  Spring Boot and Spring Data are different projects in Spring and azure-spring-boot is the azure implementation for Spring Boot, the same case for azure-spring-data.

Reply:
And the azure-spring-boot-sample-data-gremlin is a sample demonstrating how to use azure-spring-boot-starter for azure-spring-data-gremlin.